pkgs/applications/science/math/geogebra/default.nix

```diff
@@ -2,7 +2,7 @@
 
 stdenv.mkDerivation rec {
   name = "geogebra-${version}";
-  version = "5-0-472-0";
+  version = "5-0-535-0";
```

Comment:
I would expect the hash to change with a new version as well?

Reply:
Jap, I updated it.